theme_notes/static/src/scss/primary_variables.scss

```diff
@@ -129,7 +133,7 @@ $o-website-values-palettes: (
     ),
 );
 
-$o-selected-color-palettes-names: append($o-selected-color-palettes-names, 'notes-1');
+$o-selected-color-palettes-names: append($o-selected-color-palettes-names, 'default-19');
```

Comment:
Does that actually make sense? Is not `defaut-19` already in that list? If this is an issue as suspected, could you make sure to communicate that to the other designers to potentially fix in all PR? :pray:

Reply:
Indeed it's no need to append it again. I removed the line and conveyed the info to the team, thanks 🙇